src/vl-progress-bar-step.js

```diff
@@ -15,6 +15,15 @@ import { VlElement, define } from '/node_modules/vl-ui-core/dist/vl-core.js';
  * 
  */
 export class VlProgressBarStep extends VlElement(HTMLElement) {
+    /**
+     * Geeft de selector om het button element te bekomen. 
+     *
+     * @return {String}
+     */
+    static get buttonSelector() {
+        return 'button.vl-progress-bar__bullet';
+    }
```

Comment:
wordt nu dan geexposed naar iedereen? misschien toch beter gewoon in vl-wizard steken? "protected" bestaat niet in JS he ...

Reply:
Lijkt me veiliger om hier te houden, wordt voorlopig alleen gebruikt door vl-ui-wizard om de buttons te exposen in kader van vl-ui-wizard. Anders gaan we de wizard te hard koppelen aan verborgen inhoud van de vl-ui-progressbar.

Reply:
Ik zou het eventueel 'protected' kunnen maken met '_' maar de vl-wizard erft niet over van de vl-wizard-step, dus die oplossing is dan niet zo 'proper'.

Reply:
Het voelt voor mij aan als een code smell.
En gaat het niet om de step zijn eigen template te laten renderen, maar dan zonder shadow dom zodat de styling van webuniversum blijft werken?
Dan kan dit een getter zijn voor de button zelf ofzo...

Reply:
Of de template toch in de progressbar renderen, maar dat had jij gevraagd om er uit te trekken blijkbaar :-)

Reply:
Ja, omdat dat er niet thuis hoort imo.

Reply:
Het is analoog aan de setup bij vl-ui-rich-data-table, dus ik zie eigenlijk geen issues. Anders hoor ik het graag.

Reply:
> Het voelt voor mij aan als een code smell.
> En gaat het niet om de step zijn eigen template te laten renderen, maar dan zonder shadow dom zodat de styling van webuniversum blijft werken?
> Dan kan dit een getter zijn voor de button zelf ofzo...

Helaas niet mogelijk want de custom tag zou ook voor problemen zorgen in de CSS verwerking.

Reply:
Ah, ze gebruiken direct descendant selectors? Da's spijtig.